Scripts/Python/xJournalBookGUIPopup.py

```diff
@@ -212,3 +242,32 @@ def IsThereACover(self, bookHtml):
         if idx >= 0:
             return 1
         return 0
+
+    def ReadCurrentPage(self):
+        global BookJson
+        global CurrentPage
+        PtDebugPrint("xJournalBookGUIPopup ReadCurrentPage",level=kDebugDumpLevel)
+        BookJson = ptVault().findChronicleEntry(kBookChronicle)
+        if BookJson:
+            BookJson = json.loads(BookJson.getValue())
+        else:
+            BookJson = {}
+        CurrentPage = BookJson.get(self.EncodeLocPath(), -1)
+        
+    def WriteCurrentPage(self):
+        PtDebugPrint("xJournalBookGUIPopup WriteCurrentPage",level=kDebugDumpLevel)
+        global BookJson
+        if CurrentPage > -1:
+            BookJson[self.EncodeLocPath()] = JournalBook.getCurrentPage()
+        else:
+            BookJson.pop(self.EncodeLocPath(), None)
+        temp = json.dumps(BookJson)
```

Comment:
```suggestion
        temp = json.dumps(BookJson, separators=(",", ":")))
```

This will compact the json more by eliminating whitespace.